packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.tsx

```diff
@@ -67,29 +69,34 @@ const AggregationsQueriesList = ({
     void fetchItems();
   }, [fetchItems]);
 
+  const [gridHeader, listItems] = useGridHeader(items);
+
+  const renderItem: React.ComponentProps<typeof VirtualGrid>['renderItem'] =
+    React.useCallback(
+      ({ index, ...props }: { index: number }) => (
+        <ConnectedItemCard index={index} items={listItems} {...props} />
+      ),
+      [listItems]
+    );
+
   if (loading) {
     return null;
   }
 
   return (
     <>
+      <div className={header}>
+        <H2 as="h1" className={title}>
+          My queries
+        </H2>
+        <div>All my saved queries in one place</div>
+      </div>
```

Comment:
Now that the filters are added, we probably can finally drop it? Otherwise we probably want it to stay in the renderHeader, otherwise it's the only part of the screen that doesn't scroll